src/Symfony/Component/Console/Descriptor/JsonDescriptor.php

```diff
@@ -64,7 +64,7 @@ protected function describeCommand(Command $command, array $options = array())
     protected function describeApplication(Application $application, array $options = array())
     {
         $describedNamespace = isset($options['namespace']) ? $options['namespace'] : null;
-        $description = new ApplicationDescription($application, $describedNamespace);
+        $description = new ApplicationDescription($application, $describedNamespace, true);
```

Comment:
Imo. the descriptor option was nice :) anyway it's internal code.. we're good :+1: